Documentation/ApiOverview/Mail/Index.rst

```diff
@@ -117,6 +162,51 @@ mbox
 How to Create and Send Mails
 ============================
 
+Both :php:`\TYPO3\CMS\Core\Mail\MailMessage` and :php:`\TYPO3\CMS\Core\Mail\FluidEmail) inherit
+from :php:`Symfony\Component\Mime\Email` and have a similar API. **FluidEmail** is specific
+for sending emails based on Fluid.
+
+Send email with `FluidEmail`
+----------------------------
+
+This sends an email based using an existing Fluid template. Make sure the paths
+are setup as described in :ref:`mail-configuration-fluid`.
+
+.. code-block:: php
+
+   $email = GeneralUtility::makeInstance(FluidEmail::class);
+   $email
+       ->to('contact@acme.com')
+       ->from(new Address('jeremy@acme.com', 'Jeremy'))
+       ->subject('TYPO3 loves you - here is why')
+       ->setFormat('html') // only HTML mail
+       ->setTemplate('TipsAndTricks')
+       ->assign('mySecretIngredient', 'Tomato and TypoScript');
+   GeneralUtility::makeInstance(Mailer::class)->send($email);
+
+Defining a custom email subject in a custom template:
+
+.. code-block:: html
+
+   <f:section name="Subject">New Login at "{typo3.sitename}"</f:section>
+
+Building templated emails with Fluid also allows to define the language key,
+and use this within the Fluid template:
+
+.. code-block:: php
+
+   $email = GeneralUtility::makeInstance(FluidEmail::class);
+   $email
+       ->to('contact@acme.com')
+       ->assign('language', 'de');
+
+.. code-block:: html
```

Comment:
The rendered result looks like a single code block, even if two are generated. Maybe we could add a sentence between both, to improve rendered result?